spec/factories/course_assessment_submissions.rb

```diff
@@ -8,7 +8,7 @@
       auto_grade true # Used only with any of the submitted or finalised traits.
       creator
     end
-    assessment { build(:assessment, course: course) }
+    assessment { create(:assessment, :with_mcq_question, course: course) }
```

Comment:
To be fair I rather add this trait to all submissions that need an assessment with MCQ questions, since traits are additive and I prefer to be more explicit. 

I'm ok if you do it here, but if you do, you need to add the published trait as well, otherwise we are building an unpublished assessment which theoretically isn't correct either.

Reply:
Discussed offline and I think we can keep it this way since this is the more common use case and workflow.